src/FlexExampleApp.mxml

```diff
@@ -1,14 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<s:ViewNavigatorApplication xmlns:fx="http://ns.adobe.com/mxml/2009" 
-							xmlns:s="library://ns.adobe.com/flex/spark" xmlns:mx="library://ns.adobe.com/flex/mx" firstView="views.FlexExampleAppHomeView" applicationDPI="160">
-	<fx:Declarations>
-		<!-- Place non-visual elements (e.g., services, value objects) here -->
-	</fx:Declarations>
+<s:ViewNavigatorApplication
+        xmlns:fx="http://ns.adobe.com/mxml/2009"
+        xmlns:s="library://ns.adobe.com/flex/spark"
+        width="100%" height="100%"
+
+        creationComplete="onCreationComplete()"
+        firstView="views.FlexExampleAppHomeView"
+        firstViewData="{appModel}"
+        >
+
 	<fx:Script>
 		<![CDATA[
-			import com.heyzap.sdk.ads.HeyzapAds;
-		]]>
+        import com.heyzap.sdk.ads.HeyzapAds;
+
+        import model.AppModel;
+
+        [Bindable]
+        private var appModel:AppModel = new AppModel();
+
+        protected function onCreationComplete():void {
+
+            HeyzapAds.getInstance().start("9317e0b9c4eaf41ed68e907e22ab3c5a", HeyzapAds.FLAG_NONE);
```

Comment:
also, for flags, it might be useful to use the flag to disable automatic prefetching, both as an example of flag usage and so that the fetch button is the only way ads are fetched.